tests/unit/iotics/lib/identity/register/test_document_builder.py

```diff
@@ -258,11 +275,39 @@ def test_can_remove_not_existing_key_without_error(doc_did, doc_proof, doc_keys,
     assert existing_doc_key_set == new_doc_key_set
 
 
+def test_building_a_register_doc_does_not_raises_if_double_add(doc_keys):
```

Comment:
can this be renamed to
`test_building_a_register_doc_does_not_raises_if_same_key_is_added_twice`?